graphics/xfractint/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                xfractint
+version             20.04p14
+categories          graphics
+maintainers         @jflude \
+                    openmaintainer
+description         A highly versatile fractal generator
+long_description    Fractint is a freeware fractal generator created for PCs \
+    and ported to Unix/X Window. It has been in development since 1988. \
+    Fractint can also draw cellular automata and stereoscopic images.
+homepage            https://www.fractint.org/
+platforms           darwin
+license             Freeware
```

Comment:
I don't think that's a valid Macports license. See https://trac.macports.org/wiki/PortfileRecipes#license.

Reply:
Changed to "Permissive" (closest in meaning to "Copyright freeware", I think).

Reply:
That sounds reasonable.